python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -93,4 +93,348 @@ def pytest_unconfigure(config):
+    "Categorical categories cannot be null",
+    "Categorical categories must be unique",
+    "'values' must be a NumPy array, not Index",
+    "value must be a scalar",
+    "All columns must be of equal length",
+    "7 is not in range",
+    "operands could not be broadcast together with shapes (4,) (3,)",
+    "Length mismatch: Expected axis has 2 elements, new values have 1 elements",
+    "Shape of passed values is (3, 1), indices imply (2, 1)",
+    "Array conditional must be same shape as self",
+    "Could not convert strings to integer type due to presence of non-integer values.",
+    "Length of values (3) does not match length of index (4)",
+    "Inferred frequency 4D from passed values does not conform to passed frequency D",
+    "Cannot index with multidimensional key",
+    "Could not convert strings to float type due to presence of non-floating values.",
+    "storage_options passed with file object or non-fsspec file path",
+    "CategoricalColumns do not use data attribute of Column, use `set_base_children` instead",
+    "CUDF failure at: /raid/pgali/cudf/cpp/src/io/comp/uncomp.cpp:432: Failed to parse GZIP header while fetching source properties",
+    "CUDF failure at: /raid/pgali/cudf/cpp/src/io/comp/uncomp.cpp:615: Decompression: error in stream",
+    "CUDF failure at: /raid/pgali/cudf/cpp/src/io/comp/uncomp.cpp:448: Failed to parse ZIP header while fetching source properties",
+    "Resampling by DateOffset objects is not yet supported.",
+    "Fast implementation not available. Falling back to the slow implementation",
+    "freq is not implemented yet",
+    "US/Eastern must be a zoneinfo.ZoneInfo object in pandas_compatible mode.",
+    "cuDF does not yet support PeriodDtype",
+    "US/Pacific must be a zoneinfo.ZoneInfo object in pandas_compatible mode.",
+    "Expected np.datetime64 but got: double",
+    "Custom pandas ExtensionDtypes are not supported",
+    "not supported",
+    "cuDF does not yet support SparseDtype",
+    "Expected np.datetime64 but got: duration[ns]",
+    "Series with Complex128DType is not supported.",
+    "Unsupported column type passed to create an Index: <class 'cudf.core.column.decimal.Decimal128Column'>",
+    "Lists concatenation for this operation is not yetsupported",
+    "UTC must be a zoneinfo.ZoneInfo object in pandas_compatible mode.",
+    "dropna is not currently supported.",
+    "database is locked",
+    "(sqlite3.OperationalError) database is locked",
+    "Option 'chained_assignment' has already been registered",
+    "1",
+    "dtype('<m8[h]')",
+    "dtype('<m8[m]')",
+    "'tags'",
+    "nan",
+    "dtype('>M8[ms]')",
+    "'20130903'",
+    "'Level exp not found'",
+    "'Requested level (ID) does not match index name (None)'",
+    "(0, nan)",
+    "\"label=Interval(1, 4, closed='right') not in index\"",
+    "\"label=Timestamp('2019-01-01 00:00:00') not in index\"",
+    "\"label=Timedelta('1 days 00:00:00') not in index\"",
+    "7",
```

Comment:
This is too generic, no? Can we add the full error message?

Reply:
The error itself is `KeyError: 7`, nothing more than that. More information can be extracted if we inspect the stack-trace also programmatically, but that's overkill for something like this.